### DISCUSSION TITLE
Dynamic range gets worse with the top shield?

### DISCUSSION THREAD

**Discussion by njyq2008** [jankae/LibreVNA#273] [Q&A] [answered]
Hi,

I'm currently assembling a LibreVNA from scratch, strictly following the guidelines and BOM given in the project docs. I've encountered an issue that I'm hoping you might have insights into.

Initially, when I tested the PCBA without the shields, the dynamic range, as depicted in Fig.1, seemed not to be very good. To address this, I proceeded to install both the top and bottom shields, expecting an improvement in performance. To my surprise, the dynamic range deteriorated further (Fig.2) after the shields were in place (especially the top shield). I'm quite puzzled by this unexpected outcome and am seeking your expert opinions on what might be causing this peculiar phenomenon.

Any thoughts or suggestions on how to troubleshoot this issue would be greatly appreciated.

Fig.1 raw PCBA, without shields; Port1-open, Port2-open, no calibration
<img width="1920" alt="e0d9f23598c416d5925a2433fddb2b1" src="https://github.com/user-attachments/assets/642df18b-a9ce-4b3c-9ed5-29f61eb116bc">

Fig.2 PCBA, with both shields; Port1-open, Port2-open, no calibration
<img width="1920" alt="177754a0ae796e594e23bf331634c93" src="https://github.com/user-attachments/assets/0e5e51ea-07cc-49e9-b811-87f50f72bb34">

**Comment by jankae** [comment, marked as answer]
The one thing that comes to mind is the contact between the enclosure and the PCB. If it only makes contact at some small spots, you can actually decrease dynamic range because the signal can travel quite well in the very small gaps between the enclosure and the PCB. This is also the reason why the enclosure uses that many screws.

Things to check for:
- Remaining flux/solder on the exposed copper areas. All of them must be completely clean
- Burrs on the enclosure. Make sure that no edges protude beyond the mating surface of the enclosure
- Screw torque. Tighten all screws firmly

**Reply by njyq2008** [reply]
Yes, exactly true! Aftering tightening all the screws, the dynamic range improves quite a a lot and everything runs smoothly. Thank you so much.
